.github/workflows/workflow.yml

```diff
@@ -221,6 +224,7 @@ jobs:
           cloud-url: file://~
           stack-name: dev
           upsert: true
+          downsert: true
```

Comment:
```suggestion
```

Reply:
Done. I've reverted [the workflow commit](https://github.com/pulumi/actions/commit/487ad9f6f0b157d1933172f6812c07e16eefa5e0).